Formula/terraform-docs.rb

```diff
@@ -26,12 +26,16 @@ class TerraformDocs < Formula
         :revision => "c8470e45692f168e8b380c5d625327e756d7d0a9"
   end
 
+  def version
+    stable.url.split("/").last.gsub(/\.tar\.gz/, "")
+  end
+
   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/segmentio/terraform-docs").install buildpath.children
     Language::Go.stage_deps resources, buildpath/"src"
     cd "src/github.com/segmentio/terraform-docs" do
-      system "go", "build", "-o", "#{bin}/terraform-docs"
+      system "go build -o #{bin}/terraform-docs -ldflags '-X main.version=#{version}'"
```

Comment:
please keep the commas. also you should just use the preexisting `version` variable.